src/app/screens/Onboard/TestConnection/index.jsx

```diff
@@ -49,19 +164,28 @@ export default function TestConnection() {
 
             {accountInfo && accountInfo.alias && (
               <div>
-                <h1 className="text-3xl font-bold">Connection success! 🎉</h1>
+                <h1 className="text-3xl font-bold">Success! 🎉</h1>
                 <p className="text-gray-500 mt-6">
                   Awesome, you're ready to go!
-                  {faucetEnabled && accountInfo.balance === 0 && (
-                    <p>
-                      But it seems you're wallet has 0 Sats. To get started we
-                      can send you some Sats...
-                      <a href="#" onClick={claimSats}>
-                        click here.
+                </p>
+                <div>
+                  {faucetURL && accountInfo.balance === 0 && (
+                    <div className="text-gray-500">
+                      You're wallet is currently empty. &nbsp;
+                      <a
+                        href="#"
+                        className="underline"
+                        onClick={(e) => {
+                          e.preventDefault();
+                          setShowFaucet(true);
+                        }}
+                      >
+                        To get started we can send you some Sats...
                       </a>
-                    </p>
+                      {faucetURL && renderFaucet()}
```

Comment:
You're already checking if faucetURL exists at line 172, so an extra check is not necessary.